src/typings.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module "default-gateway" {
```

Comment:
There is already a type definition for this: `npm install @types/default-gateway --save-dev`

Reply:
Nice I haven't seen this. I'll update the PR